tests/dependencies/helpers.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Only for use in dependencies tests which need to test default value behavior.
+ */
+export const _negateDependencyKeyDefaultValueTestContext = () => {
```

Comment:
Would be good to have a rule of some sort to prevent people from using this gratuitously

Reply:
Sure, though I think the underscore should be enough to scare people away.